yt_dlp/extractor/firsttv.py

```diff
@@ -12,7 +12,7 @@
 class FirstTVIE(InfoExtractor):
     IE_NAME = '1tv'
     IE_DESC = 'Первый канал'
-    _VALID_URL = r'https?://(?:www\.)?1tv\.ru/(?:[^/]+/)+(?P<id>[^/?#]+)'
+    _VALID_URL = r'https?://(?:www\.)?(?:sport)?1tv\.ru/(?:[^/]+/)+(?P<id>[^/?#]+)'
```

Comment:
I know this part of the pattern is old code but I'm always gonna suggest this when I see it
```suggestion
    _VALID_URL = r'https?://(?:www\.)?(?:sport)?1tv\.ru/(?:[^/?#]+/)+(?P<id>[^/?#]+)'
```